<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendingMailWidget</class>
 <widget class="QWidget" name="SendingMailWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mail verfassen</string>
  </property>
  <widget class="QPushButton" name="sendMailPushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string>Mail Absenden</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>671</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Mail verfassen</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="senderLabel">
        <property name="text">
         <string>Absender:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="senderLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="recipientsLabel">
        <property name="text">
         <string>EmpfÃ¤nger:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="recipientsLineEdit">
        <property name="toolTip">
         <string>Hinweis: Mehrere EmpfÃ¤nger mit &quot;,&quot; trennen z.B. alice@thundermail.de,bob@thundermail.de,...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="subjectLabel">
        <property name="text">
         <string>Betreff:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="subjectLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="bodyPlainTextEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="feedbackOutput">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
